docs/source/management/monitoring/setup.md

```diff
@@ -9,29 +9,8 @@ see the [`ScyllaDBMonitoring` API reference](../../api-reference/groups/scylla.s
 
 ## Requirements
 
-Before you can set up your ScyllaDB monitoring, you need {{productName}} and a `ScyllaCluster` already installed in your Kubernetes cluster.
-For more information on how to deploy {{productName}}, see [the installation guide](../../installation/overview.md).
-
-### Deploy Prometheus Operator
-
-:::{note}
-`ScyllaDBMonitoring` relies on the Prometheus Operator for managing Prometheus-related resources.
-You can deploy it in your Kubernetes cluster using the provided third-party examples. If you already have it deployed
-in your cluster, you can skip this step.
```

Comment:
Shouldn't we leave this note (about skipping if you have it already deployed) in the installation guide?